farmdata2_modules/fd2_tabs/resources/DateSelectionComponent.js

```diff
@@ -50,8 +53,14 @@ let DateSelectionComponent = {
                 this.selectedDate = this.earliestDate;
             }
             this.$emit('date-changed', this.selectedDate)
-        }
+        },
     },
+    watch: {
```

Comment:
Should also emit 'date-changed' so that the parent page is notified when the date is changed by a change to the prop.

Reply:
Test this as well

Reply:
Emitting 'date-changed' will be handled with the existing checkBound() method to check invalid changes in the prop.